pkg/api/norman/customization/setting/setting.go

```diff
@@ -82,12 +81,6 @@ func Validator(request *types.APIContext, schema *types.Schema, data map[string]
 	case "auth-user-info-resync-cron":
 		_, err = providerrefresh.ParseCron(newValueString)
 	case "kubeconfig-token-ttl-minutes":
-		generateToken := strings.EqualFold(settings.KubeconfigGenerateToken.Get(), "true")
-		if generateToken {
-			return httperror.NewAPIError(httperror.ActionNotAvailable, fmt.Sprintf("kubeconfig-token-ttl-minutes can be set only if rancher doesn't generate token, "+
-				"disable kubeconfig-generate-token"))
-		}
-
```

Comment:
Believe we discussed this, but for posterity, why do we want to remove the "helper" here ?

Reply:
I had similar thoughts. This code only stopped users from changing the default value of `kubeconfig-token-ttl-minutes` from 16 hours not from using the `kubeconfig-token-ttl-minutes` setting. Furthermore, this is only run if users use the old U, now that setting edits are sent through steve. If we want to keep this code then I would also purpose that we need to add validation to steve.

Reply:
> However, the login token can be used regardless.
>  ...if we think people are using the token outside of using the CLI in the kubeconfig then this makes sense.

Confused by this. You mean if people are generating tokens in the CLI via the token command and outside of the login command ? And they can't change the TTL of those away from the default of 16 hours. 
Though I thought point of this red flag was so to inform people who were thinking it would add TTL to kubeconfigs generated from the UI.

> If we want to keep this code then I would also purpose that we need to add validation to steve.

That's a good point. I was thinking it was still using norman in new UI but it's not.

Overall I'd say we could clear this up with very strong UI messages or docs, and then remove the warning error. But I don't want to remove the warning without those.

Reply:
[Dashboard Issue](https://github.com/rancher/dashboard/issues/6120)
[Docs PR](https://github.com/rancher/docs/pull/4140)